### ISSUE TITLE
Stats.py not working on backus

### ISSUE THREAD

**Issue by Berrescuda** [BenningtonCS/GFS#102]
Stats.py is not being read in our dogfood setup.

**Comment by edaniszewski** [comment]
on the website:

...
2013-11-14 13:29:07,834 ERROR : Couldnt read stats file
2013-11-14 13:30:04,970 ERROR : Couldnt read stats file
2013-11-14 13:31:08,026 ERROR : Couldnt read stats file
2013-11-14 13:32:04,145 ERROR : Couldnt read stats file

so, it seems the code is getting caught

```
                                    try:
                                            file = urllib2.urlopen("http://"+ hosts[x] +":8000/httpServerFiles/stats.txt")
                                            fileData = file.read().split('|')
                                            difference = str(float(fileData[1]) - float(fileData[0]))
                                    except IOError:
                                            logging.error("Couldnt read stats file")
```

**Comment by edaniszewski** [comment]
looks like rohail fixed it 21491e3f301306faf6f7c30bf01d94b5ab4cb67c
